config/quality/checkstyle/checkstyle.xml

```diff
@@ -176,7 +176,9 @@
         <module name="AbstractClassName">
             <property name="format" value="^A[A-Z].+$" />
         </module>
-        <module name="CatchParameterName" />
+        <module name="CatchParameterName">
+            <property name="format" value="^[a-z][a-zA-Z0-9]*$" />
```

Comment:
Because I like to give exceptions a naming like e2 and e3 if I don't care at all about which specific exception it is. The alternative was to use exception names like econtactExistedException which is too long (> 15 characters) which requires to change yet another checkstyle rule